packages/app/src/systems/Chains/eth/containers/AddAssetFormDialog.tsx

```diff
@@ -0,0 +1,91 @@
+import { cssObj } from '@fuel-ui/css';
+import { Box, Button, Dialog, Icon, Input, Text, Form } from '@fuel-ui/react';
+import { useState } from 'react';
+
+import { useAssets } from '../hooks';
+
+import { store } from '~/store';
+import { shortAddress } from '~/systems/Core';
+import { useOverlay } from '~/systems/Overlay';
+
+export function AddAssetFormDialog() {
+  const { metadata } = useOverlay<{ assetAddress: string }>();
+  const { addAsset } = useAssets();
+
+  const [tokenSymbol, setTokenSymbol] = useState('');
+  const [tokenDecimals, setTokenDecimals] = useState('');
+  const [tokenImageSource, setTokenImageSource] = useState('');
+
+  return (
+    <>
+      <Dialog.Heading>
+        <Box.Flex justify="space-between" css={styles.dialogHeadingContainer}>
+          <Icon icon="ArrowLeft" onClick={store.openManageAssetsDialog} />
+          <Text color="intentsBase12" fontSize="sm">
+            Add token {shortAddress(metadata.assetAddress)}
+          </Text>
+          <Box>
+            <Dialog.Close />
+          </Box>
+        </Box.Flex>
+      </Dialog.Heading>
+      <Dialog.Description>
+        <Box.Stack>
```

Comment:
we should not need additional <Box.Stack>

Reply:
Dialog.Description does not space out the elements.  If I don't use Box.Stack I will have to add margin to all of the FormControl components should I do this?